test/integration/targets/azure_rm_devtestlab/tasks/main.yml

```diff
@@ -96,6 +99,160 @@
     that:
       - output.changed
 
+#- name: Create instance of DevTest Labs artifacts source
```

Comment:
why tests are commented out?

Reply:
no GitHub token in the CI, and  @mattclay had concerns about adding one, so we agreed to disable this particular test